### PR TITLE
[UIK-2578] added stories folder for documentation, storybook and test examples

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
Folder structure changes

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/color-picker/src/index.d.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export type ItemProps = BoxProps & {
   /**
    * Color item in hexadecimal format.
    */
-  value?: string;
+  value?: string | null;
```
Comment (ksimanov) [inline semcore/color-picker/src/index.d.ts:13]: Can this change affect users somehow? How does it related to storybook?

File: semcore/color-picker/src/index.d.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export type ColorsProps = BoxProps & {
   /**
    * Array of color items
    */
-  colors?: string[];
+  colors?: Array<string | null>;
```
Comment (ksimanov) [inline semcore/color-picker/src/index.d.ts:38]: Can this change affect users somehow? How does it related to storybook?

File: stories/components/button/__stories__/components/ButtonSizes.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Button from 'intergalactic/button';
```
Comment (ksimanov) [inline stories/components/button/__stories__/components/ButtonSizes.tsx:2]: Let's use `@semcore` here for consistency

File: stories/components/button/__stories__/components/Basic.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Button from 'intergalactic/button';
```
Comment (ksimanov) [inline stories/components/button/__stories__/components/Basic.tsx:2]: Let's use `@semcore` here for consistency

File: stories/components/color-picker/__stories__/docs-examples/input_validation.tsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ import { Flex } from 'intergalactic/flex-box';
 const Demo = () => {
   const [state, setState] = React.useState<'normal' | 'invalid'>('normal');
 
-  const onChange = (value) => {
+  const onChange = (value: any) => {
```
Comment (ksimanov) [inline stories/components/color-picker/__stories__/docs-examples/input_validation.tsx:9]: `any` doesn't look typesafe here. Is there any other way to narrower type?

File: vite.config.ts [hunk 6]
```diff
@@ -5,27 +5,27 @@ import { loadSemcoreSources } from './website/docs/.vitepress/load-semcore-sourc
...
+    pluginReact(),
+    createUnplugin<{}>(() => ({
+      name: 'semcore-resolve',
+      async resolveId(id) {
+        if (
+          !id.includes('@semcore') &&
+          !id.includes('/semcore/') &&
+          !id.startsWith('intergalactic/')
+        )
+          return null;
```
Comment (ksimanov) [inline vite.config.ts:18]: Is this one - line `if` works as expected with line break after condition?

File: semcore/color-picker/src/index.d.ts (hunk 1 above)
Comment (ilyabrower) [reply semcore/color-picker/src/index.d.ts:13]: this is real type, how we (or others) use it. 
for now, we check our code for examples with ts and that's why I added this.

File: semcore/color-picker/src/index.d.ts (hunk 2 above)
Comment (ilyabrower) [reply semcore/color-picker/src/index.d.ts:38]: this is real type, how we (or others) use it. 
for now, we check our code for examples with ts and that's why I added this.

File: stories/components/button/__stories__/components/ButtonSizes.tsx (hunk 3 above)
Comment (ilyabrower) [reply stories/components/button/__stories__/components/ButtonSizes.tsx:2]: fixed

File: stories/components/button/__stories__/components/Basic.tsx (hunk 4 above)
Comment (ilyabrower) [reply stories/components/button/__stories__/components/Basic.tsx:2]: fixed

File: stories/components/color-picker/__stories__/docs-examples/input_validation.tsx (hunk 5 above)
Comment (ilyabrower) [reply stories/components/color-picker/__stories__/docs-examples/input_validation.tsx:9]: this is not a task about 'fix types in color-picker'.
moreover, this is core problem with our typings, that's why I used `any` here

File: vite.config.ts (hunk 6 above)
Comment (ilyabrower) [reply vite.config.ts:18]: Maybe I didn't quite understand you, but I think the answer is yes.

**Review by ksimanov — APPROVED** [review]

**Pull request merged**